### DISCUSSION TITLE
求帮忙转换，vmess -----> mihomo，传输协议是httpupgrade

### DISCUSSION THREAD

**Discussion by Mitchell08** [MetaCubeX/mihomo#1687] [Q&A]
求帮忙转换，vmess -----> mihomo，传输协议是httpupgrade，用了几个订阅转换网站，都转换不了，求助大家，非常感谢了。安卓客户端v2rayng可以正常导入。但是转不了mihomo，求帮助。

vmess://eyJhZGQiOiIxMDQuMjQ3LjI1My4yMiIsImFpZCI6IjAiLCJhbHBuIjoiIiwiZnAiOiIiLCJob3N0IjoiODU2MjYzLnh5eiIsImlkIjoiYTdmZmEwMDAtODdiNy00ZTQyLTkyODYtN2JlM2M5ZDYzNTNmIiwibmV0IjoiaHR0cHVwZ3JhZGUiLCJwYXRoIjoiYmFpZHUuY29tL3VwIiwicG9ydCI6IjgwODAiLCJwcyI6ImhnIiwic2N5IjoiYXV0byIsInNuaSI6IiIsInRscyI6IiIsInR5cGUiOiItLS0iLCJ2IjoiMiJ9

![image](https://github.com/user-attachments/assets/459cd782-ccc3-44f1-bdd2-6a4921fb1655)

![image](https://github.com/user-attachments/assets/451ff28c-8617-455a-a98d-66c5f8d7b067)

**Comment by xiandu999-yu** [comment]
https://wiki.metacubex.one/config/proxies/transport/#ws-optsv2ray-http-upgrade
